knowledge-base/fileselect-disable-onselect.md

```diff
@@ -0,0 +1,72 @@
+---
+title: How to change the FileSelect button's caption and disable it during selection
```

Comment:
Please, apply title style capitalization to your title and page_title as per [our style guide](https://docs.telerik.com/style-guide/titles-and-headings#capitalization). Also, consider dropping the possessive form throughout the text - "button caption" is clear enough (not required, but recommended).